docs/UserGuide.md

```diff
@@ -154,21 +157,47 @@ Format: `help`
 
 Add a contact image for each contact.
 
-Format: `add-image INDEX ai/[PATH-TO-IMAGE]`
+Format: `add-image INDEX ai/PATH-TO-IMAGE`
 
 * Adds an image to the contact at the specified `INDEX`
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3,...
-* Ensure that `ai/` prefix is used before the image path.
-* If the image cannot be found or user did not specify a contact image, a default image will be used
-
-> **Note:** The `[PATH-TO-IMAGE]` provided must be an absolute path, and should not be provided in quotation marks.
-> For instance: `add-image 2 ai/"C:/Users/user/Downloads/weekiat.png"` will be invalid, whereas 
-> `add-image 2 ai/C:/Users/user/Downloads/weekiat.png` will be valid.
+* Ensure that `ai/` prefix is used before the image path (note that `ai/` is not part of your path)
+* Initially, a default image is used for a contact
+* After an image is added, if the application is unable to retrieve the image (e.g. erroneous manual edit of addressbook.json or the saved images) then it will revert to the default image
 
 Examples:
+* Windows: `list` followed by `add-image 2 ai/C:/Users/user/Downloads/weekiat.png` adds the image `weekiat.png` to the 2nd person in the address book
```

Comment:
add in specification to the commands for different device to make it more user friendly. A good instruction is really essential for users as it can make it more clear to them. As for developers, it can help them to reduce the amount of bugs in the event if the project gets more complex and it would be harder for us to keep track